README.md

```diff
@@ -303,6 +303,13 @@ you're not using `nvm`, make sure the string you prepend to the `$PATH` variable
 contains the directory or directories that contain the `bower` and `npm`
 executables.
 
+#### For faster deployments
+Place the following in your deploy/<environment>.rb 
+```ruby
+set :linked_dirs, %w{frontend/node_modules frontend/bower_components}
```

Comment:
@dchersey thanks for adding this information!

`frontend` is the default name of the Ember application, but it's an arbitrary value that can be configured.

Would you mind updating this text to account for that?